pkg/geoipupdate/config.go

```diff
@@ -132,6 +143,10 @@ func NewConfig( //nolint: gocyclo // long but breaking it up may be worse
 		config.RetryFor = 5 * time.Minute
 	}
 
+	if _, ok := keysSeen["Parallelism"]; !ok {
+		config.Parallelism = 1
+	}
```

Comment:
I was going to suggest that we just set the default when we create `config` above, but I guess this is how we do it for `RetryFor`. Maybe there is a reason, but it isn't immediately clear to me.